src/main/java/com/easypost/service/CreditCardService.java

```diff
@@ -0,0 +1,85 @@
+package com.easypost.service;
```

Comment:
As mentioned on the model, let's remove the `CreditCard` class and logic completely and call it out in the CHANGELOG.